src/modules/claims/claims.service.ts

```diff
@@ -1494,12 +1501,36 @@ export class ClaimsService {
     if (!proofVerified) {
       errors.push(ERROR_MESSAGES.PROOF_NOT_VERIFIED);
     }
+    // Date.now() and JWT expiration time both identify the time elapsed since January 1, 1970 00:00:00 UTC
+    const isExpired = payload?.exp && payload?.exp * 1000 < Date.now();
+    if (isExpired) {
+      errors.push(ERROR_MESSAGES.CREDENTIAL_EXPIRED);
+    }
     return {
       errors: errors,
-      isVerified: !!proofVerified && !!issuerVerified,
+      isVerified: !!proofVerified && !!issuerVerified && !isExpired,
```

Comment:
`issuerVerified` seem to be boolean
```suggestion
      isVerified: !!proofVerified && issuerVerified && !isExpired,
```